feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/TimetableScreen.kt

```diff
@@ -197,8 +197,12 @@ private fun TimetableScreenPreview_List() {
                 timetable = TimetableUiState.ListTimetable(
                     timetableListUiStates = mapOf(
                         DroidKaigi2025Day.ConferenceDay1 to TimetableListUiState(
-                            persistentMapOf(),
-                            Timetable.fake(),
+                            timetableItemMap = persistentMapOf(
+                                *TimetableListUiState.TimeSlot.fakes().map { slot ->
+                                    slot to listOf(TimetableItem.Session.fake())
+                                }.toTypedArray(),
+                            ),
```

Comment:
[nitpick] Maybe we could simplify this code a bit like this 👀 
```kotlin
timetableItemMap = TimetableListUiState.TimeSlot.fakes().associateWith { 
    listOf(TimetableItem.Session.fake()) 
}.toPersistentMap(),
```

What do you think?

Reply:
@kitakkun 
I think the code you suggested is simpler and better! 💯 
The following commit addresses this issue. 🫡
https://github.com/DroidKaigi/conference-app-2025/pull/262/commits/39dbfee2e8bf97e94563df23d54182a8d33202d0